docs/source/examples/Detecting Events.mystnb

```diff
@@ -0,0 +1,304 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Detecting Events
+The procedure to track an event during an orbit's propagation is fairly simple:
```

Comment:
It would be interesting to develop a little bit more about events. You can use the introduction from your proposal 😄